### PR TITLE
fixed user services

### PR DESCRIPTION
**Pull request by YuvalOfek1**

### CODE REVIEW

File: src/services/user.services.js [hunk 1]
```diff
@@ -1,57 +1,51 @@
 import {User} from '../models/userScheme'
 
-const getAllUsers = async () => {
-    return await User.find();
+const getAll = async () => {
```
Comment (TalMekler) [inline src/services/user.services.js:3]: add docs

File: src/services/user.services.js [hunk 2]
```diff
@@ -1,57 +1,51 @@
...
+    const users = await User.find();
+    if(users){
+        return users;
+    }
+    console.log("No users found in getAll");
 }
 
-const getUserById = async (id) => {
+const getById = async (id) => {
```
Comment (TalMekler) [inline src/services/user.services.js:11]: add docs

File: src/services/user.services.js [hunk 3]
```diff
@@ -1,57 +1,51 @@
...
     return await User.findById(id);
 }
 
-const getUserByEmail = async (email) => {
+const getByEmail = async (email) => {
```
Comment (TalMekler) [inline src/services/user.services.js:15]: add docs

File: src/services/user.services.js [hunk 4]
```diff
@@ -1,57 +1,51 @@
...
     return await User.findOne({email});
 }
 
-const addUser = async (firstName, lastName, email, password, profilePic, phoneNumber, address, role, orders, numOfOrders, creditCardNumber) =>{
+const add = async (user) => {
```
Comment (TalMekler) [inline src/services/user.services.js:19]: add docs

File: src/services/user.services.js [hunk 5]
```diff
@@ -1,57 +1,51 @@
...
+        }
+    } catch(err){
+        console.log(err);
     }
+    
 }
 
-const removeUser = async (email) => {
-    await User.deleteOne({email});
+const removeById = (id) => {
```
Comment (TalMekler) [inline src/services/user.services.js:31]: add docs

File: src/services/user.services.js [hunk 6]
```diff
@@ -1,57 +1,51 @@
...
+    try{
+        User.deleteOne(_id==id);
```
Comment (TalMekler) [inline src/services/user.services.js:33]: try deleteById(id)

File: src/services/user.services.js [hunk 7]
```diff
@@ -1,57 +1,51 @@
...
-            profilePic,
-            phoneNumber,
-            address,
-            role,
-            orders,
-            numOfOrders,
-            creditCardNumber
-
-        })
+const update = (id, user) =>{
```
Comment (TalMekler) [inline src/services/user.services.js:39]: add docs

File: src/services/user.services.js (hunk 6 above)
Comment (idanasayag0) [reply src/services/user.services.js:51]: User.deleteOne(_id: id);
its done?

**Pull request closed without merging**